### PR TITLE
Remove address field and its usages

### LINKED ISSUE #53: Remove address field

### PR DESCRIPTION
**Pull request by achl1012**
Closes #53

### CODE REVIEW

File: src/test/java/seedu/address/logic/parser/EditCommandParserTest.java [hunk 1]
```diff
@@ -1,11 +1,7 @@
 package seedu.address.logic.parser;
 
 import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
-import static seedu.address.logic.commands.CommandTestUtil.ADDRESS_DESC_AMY;
-import static seedu.address.logic.commands.CommandTestUtil.ADDRESS_DESC_BOB;
 import static seedu.address.logic.commands.CommandTestUtil.EMAIL_DESC_AMY;
-import static seedu.address.logic.commands.CommandTestUtil.EMAIL_DESC_BOB;
```
Comment (SlothyCat) [inline src/test/java/seedu/address/logic/parser/EditCommandParserTest.java:7]: Was the email description of Bob not needed in the upcoming test function?

File: src/test/java/seedu/address/logic/parser/EditCommandParserTest.java [hunk 2]
```diff
@@ -180,19 +166,19 @@ public void parse_multipleRepeatedFields_failure() {
         assertParseFailure(parser, userInput, Messages.getErrorMessageForDuplicatePrefixes(PREFIX_PHONE));
 
         // mulltiple valid fields repeated
-        userInput = targetIndex.getOneBased() + PHONE_DESC_AMY + ADDRESS_DESC_AMY + EMAIL_DESC_AMY
-                + TAG_DESC_FRIEND + PHONE_DESC_AMY + ADDRESS_DESC_AMY + EMAIL_DESC_AMY + TAG_DESC_FRIEND
-                + PHONE_DESC_BOB + ADDRESS_DESC_BOB + EMAIL_DESC_BOB + TAG_DESC_HUSBAND;
```
Comment (SlothyCat) [inline src/test/java/seedu/address/logic/parser/EditCommandParserTest.java:185]: I believe this is the part where we made use of Bob's email description. Was the removal on purpose to keep the test shorter but still valid? 🤔

**Review by SlothyCat — CHANGES_REQUESTED** [review]
Removal was clean and looks good. 😆  However, there is a small part where email description was removed. Was this intended? 🤔

File: src/test/java/seedu/address/logic/parser/EditCommandParserTest.java (hunk 2 above)
Comment (achl1012) [reply src/test/java/seedu/address/logic/parser/EditCommandParserTest.java:185]: I think I removed Bob's email description by accident, thank you for pointing that out!

**Review by SlothyCat — APPROVED** [review]
LGTM! 😄

**Pull request merged**